changelog/unreleased/issue-2319

```diff
@@ -0,0 +1,12 @@
+Bugfix: Correctly dump directories into tar files
+
+The dump command previously wrote directories in a tar file in a way which
+can cause compatibility problems. This caused for example 7zip on Windows
+to open tar files containing directories. In addition it was not possible
```

Comment:
To NOT open tar files, I suppose? Also commas around ", for example," and comma after "In addition," (one more below).

Reply:
I agree